src/vs/platform/native/common/native.ts

```diff
@@ -67,6 +67,8 @@ export interface ICommonNativeHostService {
 	unmaximizeWindow(): Promise<void>;
 	minimizeWindow(): Promise<void>;
 
+	setWindowMinimumWidth(minimumWidth: number): Promise<void>;
```

Comment:
I think I would rather have a `setWindowMinimumSize` method so that the API would allow for changing the height as well and then we nicely match the electron API. I would allow the width or height to be `undefined` as a sign to not change it.